src/main/java/fi/csc/avaa/smear/parameter/TimeSeriesQueryParameters.java

```diff
@@ -0,0 +1,83 @@
+package fi.csc.avaa.smear.parameter;
+
+import fi.csc.avaa.smear.validation.ValidIsoDate;
+import fi.csc.avaa.smear.validation.ValidTimeSeriesQueryParameters;
+import lombok.Getter;
+import lombok.Setter;
+import org.eclipse.microprofile.openapi.annotations.parameters.Parameter;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.ws.rs.QueryParam;
+import java.util.List;
+
+@Getter
+@Setter
+@ValidTimeSeriesQueryParameters
+public class TimeSeriesQueryParameters {
+
+    @Parameter(description = "Name of the database table where variable data is stored in the smear database. " +
+            "If you want to select from multiple tables, please use multiple tablevariable parameters. " +
+            "Table names can be queried from the tablemetadata endpoint by a variable's table id. " +
+            "Table ids of variables can be found from the variablemetadata endpoint.",
+            example = "HYY_META")
+    @QueryParam("table")
```

Comment:
Description texts could maybe be stored as constants at some point, especially the repeating sentences and use them with string interpolation. Don't know if it would make the code less readable though. In the end its really minor issue.